src/gui/main.cpp

```diff
@@ -80,6 +85,12 @@ MyFrame::MyFrame(wxWindow* parent, wxWindowID id, const wxString& title, const w
     
 	notebook_1_Logactivity->SetSizer(sizer_1);
 	notebook_1_pane_1->SetSizer(sizer_2);
+
+	stray = new MyTaskBarIcon();
+	stray->SetIcon(wxICON(sample), wxString::FromUTF8("netvfy-agent"));
```

Comment:
@si74 so here `wxICON(sample)` is where it will search for "sample_xmp" which is defined in "sample.xpm" file.

Reply:
ahh gotcha kk!